src/patterns/equality-information/sexual-orientation/index.njk

```diff
@@ -9,8 +9,8 @@ layout: layout-example.njk
 
 {% set sexualOrientationHtml %}
   {{ govukInput({
-    id: "other-description",
-    name: "other-description",
+    id: "other_description",
```

Comment:
```suggestion
    id: "other-description",
```

Reply:
Dang, I missed one! Your suggestion changes the ID though, not the name. I think we'd agreed to leave the IDs using hyphens, as they don't cause any issues?